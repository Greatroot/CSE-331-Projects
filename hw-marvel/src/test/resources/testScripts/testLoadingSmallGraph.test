# Tests loading a graph from a small dataset.

LoadGraph smallGraph staffSuperheroes.tsv
ListNodes smallGraph
ListChildren smallGraph Ernst-the-Bicycling-Wizard
ListChildren smallGraph Notkin-of-the-Superhuman-Beard
ListChildren smallGraph Perkins-the-Magical-Singing-Instructor
ListChildren smallGraph Grossman-the-Youngest-of-them-all