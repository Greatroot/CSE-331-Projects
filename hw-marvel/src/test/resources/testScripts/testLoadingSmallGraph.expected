# Tests loading a graph from a small dataset.

loaded graph smallGraph
smallGraph contains: Ernst-the-Bicycling-Wizard	Grossman-the-Youngest-of-them-all Notkin-of-the-Superhuman-Beard Perkins-the-Magical-Singing-Instructor
the children of Ernst-the-Bicycling-Wizard in smallGraph are: Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Notkin-of-the-Superhuman-Beard(CSE403) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Notkin-of-the-Superhuman-Beard in smallGraph are: Ernst-the-Bicycling-Wizard(CSE331) Ernst-the-Bicycling-Wizard(CSE403) Grossman-the-Youngest-of-them-all(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331)
the children of Perkins-the-Magical-Singing-Instructor in smallGraph are: Ernst-the-Bicycling-Wizard(CSE331) Grossman-the-Youngest-of-them-all(CSE331) Notkin-of-the-Superhuman-Beard(CSE331)
the children of Grossman-the-Youngest-of-them-all in smallGraph are: Ernst-the-Bicycling-Wizard(CSE331) Notkin-of-the-Superhuman-Beard(CSE331) Perkins-the-Magical-Singing-Instructor(CSE331)